libs/printing/src/render.tsx

```diff
@@ -28,7 +28,8 @@ export interface PaperDimensions {
 export const PAPER_DIMENSIONS = {
   // Width of exactly 8in results in 1-3 dots of overflow. The overflowing dots print on a line of
   // their own, followed by a mostly blank line. This causes stripes in the printed page.
-  Bmd150: { width: 7.975, height: 13.25 },
+  Custom8x13pt25: { width: 7.975, height: 13.25 },
+  Custom8x11: { width: 7.975, height: 11 },
```

Comment:
Just reemphasizing this point in the PR description:

> Note that I haven't touched the logic for compacting content when it can't fit on a single page. That logic should still work as it tries compacting two steps beyond the starting layout. The starting layout is chosen given the number of contests on the ballot, so technically, we should update that logic to account for 11" vs. 13.25". But the two compacting steps should still cover most cases, and we're already considering layout adjustments for other cert issues so I didn't want to preemptively calculate new baselines.